content/en/users/tutorials/filetransfer-gridstorage/_index.md

```diff
@@ -8,4 +8,229 @@ description: >
   Use EGI File Transfer to handle data in grid storage
 ---
 
-TBD
+<!-- Adjust page weight to insert this tutorial in the right place, then remove this -->
+
+## Overview
+
+This tutorial describes how to perform a transfer usin FTS transfers services
```

Comment:
The name of the EGI Service is EGI Data Transfer, this should be used/replaced/mentioned. (also probably in the directory name like `filetransfer-gridstorage` replaced by `data-transfer-grid-storage`.

Reply:
I've changed this sentence with:

This tutorial describes the EGI Data Transfer using FTS transfers services and
WebFTS.

Reply:
Can you also please rename the folder structure as it will be the URL?
`content/en/users/tutorials/filetransfer-gridstorage/_index.md` => `content/en/users/tutorials/data-transfer-grids-torage/_index.md`